windows-server-container-tools/CleanupContainerHostNetworking/README.md

```diff
@@ -1,43 +1,58 @@
+## Tracing: Capture logs for reproducible behavior
+To capture logs for a specific, reproducible behavior/issue, run this script with the `-CaptureTraces` argument:
+```
+PS C:\> .\WindowsContainerNetworking-LoggingAndCleanupAide.ps1 -CaptureTraces
+```
+When the script is run with this option, it will give you the following prompt: `Please reproduce issues for troubleshooting now. After completing repro steps, press any key to continue...`. **When this prompt appears, reproduce the behavior/issue that you would like to capture**. Then, after reproducing the issue, **press any key to continue/end the script.**
 
-## Option 2: Run `CleanupContainerHostNetworking.ps1` with Force Networking Cleanup option
+## Host Cleanup: Remove/reset container networking components on your host
+This script can be used to refresh the container-related network components on your host. To perform a host network cleanup, run this script with the `-Cleanup` option:
+```
+PS C:\> .\WindowsContainerNetworking-LoggingAndCleanupAide.ps1 -Cleanup
+```
+*When run with this option, the script will:*
+- Stop/Remove all containers on the host, regardless of their state 
+- Remove all container networks on the host
 
-```none
-PS C:\> .\CleanupContainerHostNetworking.ps1 -ForceCleanup
+In addition to the basic `-Cleanup` option, there is also the `-ForceDeleteAllSwitches` option. *Use these options together to force an extended host cleanup*, in which Switch/NIC registry keys are removed from your host, network adapters are unbound and the host default NAT network is removed, and the HNS.data file is deleted to remove all current HNS configurations.
+```
+PS C:\> .\WindowsContainerNetworking-LoggingAndCleanupAide.ps1 -Cleanup -ForceDeleteAllSwitches
 ```
+## Assumptions/System Requirements - *READ BEFORE RUNNING SCRIPT*
+
+**WARNING:** This script requires that the host machine is not running in swarm mode. The script will make sure the host is in an inactive swarm state. If the host is in swarm mode, the script will provide an option to leave swarm mode.
+
+This script assumes the following about your system. Ensure these requirements are met before running this script on your system:
+- Docker (Docker CE, Docker EE, or Docker for Windows) is installed *and running* 
+- Microsoft Hyper-V role is enabled 
+- The Windows Containers feature is installed
+
+## \*Info/logs collected for capturing container host state
+*Whether it is run without arguments, or with any of the arguments above, as part of its basic logging functionality this script captures:*
+- The Host Network Service (HNS) event log (`c:\Windows\System32\winevt\Logs\Microsoft-Windows-Host-Network-Service-Admin.evtx`)
+- The System event log (`c:\Windows\System32\winevt\Logs\System.evtx`)
```

Comment:
For now I believe we expect people who don't know how to read these to just forward them on to us to help with troubleshooting--right now they're more for us. I'll put this on my backlog for potential future changes, however.